src/components/documentation/edit-tabs/edit-page-fetcher.component.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { IScope } from 'angular';
+
+interface IPageScope extends IScope {
+  fetcherJsonSchema: any;
+}
+
+const EditPageFetcherComponent: ng.IComponentOptions = {
+  bindings: {
+    fetchers: '=',
+    page: '=',
+    title: '<',
+  },
+  template: require('./edit-page-fetcher.html'),
+  controller: function ($scope: IPageScope) {
```

Comment:
It would be better to use a class instead of a function like I did in https://github.com/gravitee-io/gravitee-management-webui/pull/1611/files#diff-9585b3611c678fbd4aaa23eea16a2ca7e4571f8a2beb9a348da4e21fd7216b49R18